cvm/quantization/transformer.py

```diff
@@ -65,14 +70,23 @@ def quantize(self, no_realize=False):
             self._lgr.error("Please calibrate thresholds first.")
             assert False
 
-        qsym, qparams = quantize(self._sym, self._prm,
+        self._qsym, self._qprm = quantize(self._sym, self._prm,
                 self.th_dict, self.precs, self.scales, self.op_input_precs)
-        '''
-        if not no_realize:
-            qsym, qparams = self._realize()
-        qext = self._get_ext()
-        '''
-        return qsym, qparams
+        self._qext = self._get_ext()
+        return self._qsym, self._qprm, self._qext
+
+    def dump(self, fname="test_dump", directory="~/tvm-cvm/data/"):
+        assert self._qsym is not None
+        import os
+        directory = os.path.expanduser(directory)
+        qsym_path = os.path.join(directory, fname+'.json')
+        qprm_path = os.path.join(directory, fname+'.params')
+        qext_path = os.path.join(directory, fname+'.ext')
```

Comment:
use the `extend_fname` function in `utils.py`

Reply:
fixed